src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -84,11 +84,6 @@ public function testUnsilencing()
 
     public function testStacking()
```

Comment:
hum in fact it would require this test case to be remove in 3.4 altogether because DebugClassLoader shouldn't call these methods anymore then. Even better :)